src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form form--big"
+      action="#"
+      method="get"
+      data-qa="keypress"
```

Comment:
Read [requirements](https://github.com/mate-academy/layout_search-bar-airbnb#requirements) carefully about where to place `data-qa` attributes